ui/src/app/pages/signed-in/component.ts

```diff
@@ -95,7 +95,7 @@ export class SignedInComponent implements OnInit, OnDestroy, AfterViewInit {
         this.profile = profile as unknown as FetchProfile;
         setInstitutionCategoryState(this.profile.verifiedInstitutionalAffiliation);
         if (hasRegisteredAccess(this.profile.accessTierShortNames)) {
-          cdrVersionsApi().getCdrVersions().then(resp => {
+          cdrVersionsApi().getCdrVersionsByTier().then(resp => {
```

Comment:
This is the change that drives everything else: this call returns a `CdrVersionTiersResponse` which includes Controlled Tier instead of `CdrVersionListResponse` which does not.